### PR TITLE
[PMSA-101] Fixed bundling of AB configs for getVariant()

### PR DESCRIPTION
**Pull request by dan-gant-zocdoc**
@Andrew-Hagedorn-ZocDoc @douglas-longnecker-zocdoc @thomas-huston-zocdoc

### CODE REVIEW

**Comment by thomas-huston-zocdoc** [comment]
@dan-gant-zocdoc can you add jasmine tests for this? https://github.com/ZocDoc/Bundler/blob/master/src/jasmine-tests/bundle-stats/bs-add-ab-config-file-spec.js

File: src/jasmine-tests/bundle-stats/bs-add-ab-config-file-spec.js [hunk 1]
```diff
@@ -32,11 +32,14 @@ describe("BundleStatsCollector - ", function() {
...
+  var getAbVariant = function(ab) {
+          return "AB.getVariant('" + ab + "')";
+      },
       ab1 = getAbConfig(string1),
       ab2 = getAbConfig(string2),
-      ab3 = getAbConfig(string3);
+      ab3 = getAbVariant(string3);
 
-  it("Adds ab configs to the collection.", function() {
+  it("Adds ab configs and variants to the collection.", function() {
```
Comment (dan-gant-zocdoc) [inline src/jasmine-tests/bundle-stats/bs-add-ab-config-file-spec.js]: @thomas-huston-zocdoc I updated the test to cover AB variants. The README.MD suggests I need to install jasmine-node to run these tests. Is that accurate or is there an easier way? Can I run this repo through integration?

Comment (thomas-huston-zocdoc) [inline src/jasmine-tests/bundle-stats/bs-add-ab-config-file-spec.js]: That's accurate, you'll need jasmine-node. There's no Bundler CI currently.

**Comment by dan-gant-zocdoc** [comment]
AB bundling tests passed

**Pull request merged**